doc/Witch/intro-to-witch.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introduction to Witch
+---
```

Comment:
I think we normally do # as the title? Do we want to do that here as well for consistency?
```suggestion
# Introduction to Witch
```

Reply:
Sorry, my env auto-gen markdown, and that's usually the format static generators use.